apps/tup-ui/src/pages/ProjectView/ProjectView.tsx

```diff
@@ -6,18 +6,25 @@ import {
 } from '@tacc/tup-components';
 import React from 'react';
 import { Outlet, useParams } from 'react-router-dom';
+import styles from './ProjectView.module.css';
 
 const ProjectView: React.FC = () => {
   const { projectId } = useParams<{ projectId: string }>();
   if (!projectId) return <div>No project selected.</div>;
 
   return (
     <RequireAuth>
-      <PageLayout
-        top={<ProjectHeader projectId={parseInt(projectId)} />}
-        left={<UserList projectId={parseInt(projectId)} />}
-        right={<Outlet />}
-      ></PageLayout>
+      <section className={styles['project-view']}>
+        <PageLayout
+          top={<ProjectHeader projectId={parseInt(projectId)} />}
+          left={<UserList projectId={parseInt(projectId)} />}
+          right={
+            <div>
+              <Outlet />
+            </div>
```

Comment:
Unless these instructions result a failure I did not catch, please:

1. Remove this extra `<div>`.
1. Change `.pub-details-container`'s `margin` to `padding` (and maybe increase its `min-width`).
    <sup>(Margin is outside the container, thus outside the parent grid. Padding is inside the container, thus also inside the parent grid.)</sup>
    - To preserve the rendered minimum width, change `min-width` from `500px` to `545px`.
        <sup>(The space inside as padding instead of margin reduces box width... I guess.[^1])</sup>
2. Change `.project-view div` selector to `.project-view > div > div` (to not affect `.pub-details-container`).
    <sup>(The `overflow: auto` [triggers](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context) triggers a [BFC](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout/Intro_to_formatting_contexts) which I think is what shrink-wraps the `.pub-details-container` if `overflow: auto`.)</sup>



[^1]: The padding is 35px, but the width shrunk by 45px, so my theory is incomplete and maybe wrong.

Reply:
The commit 0a215c2 has most of these changes.

The padding bottom is not showing though.

Even when I fixed `.project-view div` selector changed to `.project-view > div` instead of `.project-view > div > div`, the padding still did not show.

I am ignoring the bug (and the extra `<div>`) for now.